### DISCUSSION TITLE
Add Firework Explosion Power Modifier

### DISCUSSION THREAD

**Discussion by satadrive420** [PurpurMC/Purpur#1307] [__Dismissed__] [answered]
Hello all,

I see there's a modifier for end crystal and respawn anchor explosions, and I think adding one for firework rockets would also be cool. 
It would also be cool if there was an option for fireworks to do block damage.

**Comment by RhythmicSys** [comment, marked as answer]
This would be better as a plugin or datapack - datapack for damage modifiers and plugin for doing block damage.
